### PR TITLE
updated the about section and the subtitle of the landing page as per…

### PR DESCRIPTION
**Pull request by ksaini75**
… client's request

### CODE REVIEW

File: src/components/AboutPage/AboutPage.js [hunk 1]
```diff
@@ -113,10 +113,13 @@ const AboutPage = () => {
           <Row id="about-section">
             <Col className="mt-5" md={6} id="first">
               <div>
-                <h1 className="about-title text-center mb-4" >About Me</h1>
+                <h1 className="about-title text-center mb-4" >About KCElectricalContractorLLC</h1>
               </div>
```
Comment (ksaini75) [inline src/components/AboutPage/AboutPage.js:117]: Client requested to change "About Me" to "About KCElectricalContractorLLC"

File: src/components/AboutPage/AboutPage.js [hunk 2]
```diff
@@ -113,10 +113,13 @@ const AboutPage = () => {
...
               <div>
-                <p className="about-text text-center mb-5" >I have 8 years in residential and commercial and also know how to troubleshoot and also know how to install panels how to wire trace install new breakers and how to install ceiling fans sconces light in new and old construction services ceiling fan and new and old works.
+                <p className="about-text text-center mb-5" >We are glad to offer you a wide range of residential electrical services - from changing fixtures, upgrading panels, garden lighting to outdoor outlets and much more.
```
Comment (ksaini75) [inline src/components/AboutPage/AboutPage.js]: I added a break tag to add some spaces between the paragraphs

File: src/components/LandingPage/LandingPage.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ const LandingPage = () => {
             <div className="main-img mb-5" id="home">
                 <div className="container text-center">
                     <p className="company-name text-center">KCElectricalContractorLLC</p>
-                    <p className="company-tag">Company is Insured</p>
+                    <p className="company-tag">Company is Insured and Licensed</p>
                     <button className="company-btn" onClick={onBtnClick}>Request An Estimate</button>
```
Comment (ksaini75) [inline src/components/LandingPage/LandingPage.js:21]: Client requested to add "and Licensed"

**Comment by MrGDS2** [comment]
Everything looks good!

**Pull request merged**